pages/banks/add.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <iframe
     class="airtable-embed h-full min-h-screen"
-    src="https://airtable.com/embed/shrARSrEwz4411Gg5?backgroundColor=gray"
+    src="https://airtable.com/embed/shrARSrEwz4411Gg5?backgroundColor=grey"
```

Comment:
That's not a tailwind class. It's a src parameter for airtable iframe. Please revert all.

Reply:
Done. See next commit